sunbeam-python/sunbeam/commands/bootstrap.py

```diff
@@ -131,13 +137,27 @@ def bootstrap(
     roles: List[Role],
     topology: str,
     database: str,
+    manifest: Optional[Path] = None,
     preseed: Optional[Path] = None,
     accept_defaults: bool = False,
 ) -> None:
     """Bootstrap the local node.
 
     Initialize the sunbeam cluster.
     """
+    client: Client = ctx.obj
+
+    # Validate manifest file
+    manifest_obj = None
+    if manifest:
+        manifest_obj = Manifest.load(
+            client, manifest_file=manifest, include_defaults=True
+        )
+    else:
+        manifest_obj = Manifest.get_default_manifest(client)
```

Comment:
If I do a first bootstrap with manifest, and then re-run bootstrap without giving a manifest, what's the behavior ?

Reply:
In this case, the bootstrap re-run will pick manifest with default values rather than user provided.
If the user re-runs bootstrap, it will be good if user uses the same options provided during the first run.